src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -57,11 +68,12 @@ void createLine2() {
     @Test
     void getLines() {
         // given
+
         // 지하철_노선_등록되어_있음
-        Map<String, String> params1 = makeParam("bg-red-600", "신분당선");
+        LineRequest params1 = makeParam("bg-red-600", "신분당선", 1L,2L, 10L);
```

Comment:
변수명을 의미 있는 단어로 바꾸면 어떨까요?

```suggestion
        LineRequest 신분당선 = makeParam("bg-red-600", "신분당선", 1L,2L, 10L);
```